### ISSUE TITLE
RESIZE_NEAREST_NEIGHBOR - Operation version not supported during compile

### ISSUE THREAD

**Issue by Xoisai** [google-coral/edgetpu#420]
I'm having issues with compilation on a model, when compiling, all processes are mapped okay with the exception of resize operations.

Using:
 - Tensorflow 2.5.0
 - Tensorflow-model-optimization 0.6.0
 - Browser Edge TPU Compiler version 15.0.340273435

Compiling quantised tflite model leads to the following:

```
Edge TPU Compiler version 15.0.340273435
Input: test-20210714-181902.tflite
Output: test-20210714-181902_edgetpu.tflite

Operator                       Count      Status

...
RESIZE_NEAREST_NEIGHBOR        4          Operation version not supported
```

Any suggestions?

**Comment by hjonnala** [comment]
please check out edgetpu supported operations and limitations for RESIZE_NEAREST_NEIGHBOR   here: https://coral.ai/docs/edgetpu/models-intro/#supported-operations

Any unsupported operations would be mapped and  performed on CPU instead of EdgeTPU.

**Comment by Xoisai** [comment]
Thanks @hjonnala, I checked that before and it should be okay, is there a way to override the loss aversion and map anyway?

**Comment by hjonnala** [comment]
can you please share your tflite model to check with the internal team.

**Comment by Xoisai** [comment]
Of course, [here](https://drive.google.com/file/d/1vTayquHmWzQdpXIhOzCcQ2jkRewWfN3M/view?usp=sharing) it is. Thank you for your help!

**Comment by hjonnala** [comment]
Sorry, I was referring to uncompiled tflite model. Please share uncompiled model.

**Comment by Xoisai** [comment]
Of course, [this](https://drive.google.com/file/d/1aL8DSP6GWzSRSV94Z8FXgIS7Q3WCKppO/view?usp=sharing) is the architecture. I was having difficulties with concatenation layers when converting to quantized tflite, hence the quantization annotations, and generating the final model for tflite conversion with:

```
    with quantize_scope({'NoOpQuantizeConfig':  default_8bit_quantize_configs.NoOpQuantizeConfig}):
        quantized_model = quantize_model(model)
```

The tflite model was then saved with:

```
    converter = tf.lite.TFLiteConverter.from_keras_model(model)
    converter.optimizations = [tf.lite.Optimize.DEFAULT]
    converter.inference_input_type = tf.uint8
    converter.inference_output_type = tf.uint8
    quantized_tflite_model = converter.convert()
```

**Comment by hjonnala** [comment]
Hi,

Current compiler only works for TF1.x RESIZE_NEAREST_NEIGHBOR

The support for TF2.x RESIZE_NEAREST_NEIGHBOR  will be added in our next release and It is going to release very soon. May be within month.

**Comment by Xoisai** [comment]
Okay thank you for looking into it, I'll have a go with the 1.15 Lite converter.

**Comment by hjonnala** [comment]
@Xoisai support for RESIZE_NEAREST_NEIGHBOR is added in new version of edgetpu compiler. But I see not all of the operations  mapped to Edge TPU.

![image](https://user-images.githubusercontent.com/86132058/127183666-21e0d05e-903e-4666-a837-5a83c82a4337.png)

**Comment by Xoisai** [comment]
@hjonnala It needs the -sa argument, however with this included, 2 RESIZE_NEAREST_NEIGHBOUR ops are still not mapped, as with your compilation:

![Screenshot 2021-07-27 at 16 50 21](https://user-images.githubusercontent.com/48554828/127185977-d061869d-1e02-472e-8cc2-00f4b1de3d34.png)

**Comment by hjonnala** [comment]
Yes, it is. It is  recommended not to use -a flag as this flag is still experimental.

Just curious out of 24 conv_2d layers there are 16 mapped to edge tpu and 8 are not mapped. May I ask is there any deference in those layers?

**Comment by Xoisai** [comment]
The layers (see model [here](https://drive.google.com/file/d/1BBqmoqAjfpsgdRRVoxXzqHnj_HFQZThx/view?usp=sharing)) vary in filter dimension and kernel size, with some upsampling and concatenation - could it be the concatenated Conv2D layers causing the issue? 

The concatenation ops are currently optimised for quantisation aware training, but I've tried to compile with a non-QAT, post training full-integer optimised model on compiler version 16.0.384591198 and I'm getting the same result for Conv2D (with only -s flag):

![Screenshot 2021-07-27 at 18 42 50](https://user-images.githubusercontent.com/48554828/127202166-31681775-b132-4dcb-b707-e295bfaa0577.png)

**Comment by hjonnala** [comment]
@Xoisai two of the RESIZE_NEAREST_NEIGHBOR operations are not mapping to edgetpu due to the size of operation.

**Comment by hjonnala** [comment]
As of now, the only way to map all RESIZE_NEAREST_NEIGHBOR operations to edgetpu is by reducing their size. May be [this strategy](https://github.com/google-coral/edgetpu/issues/461#issuecomment-913096712) is helpful.
